src/index.js

```diff
@@ -0,0 +1,250 @@
+import { parseBearer } from './helpers/tokenFormatter';
+import {
+  ENVIRONMENT_IS_REACT_NATIVE,
+  ENVIRONMENT_IS_NODE,
+  ENVIRONMENT_IS_WEB,
+  ENVIRONMENT_IS_WORKER,
+  ERROR_REFRESH_TOKEN_EXPIRED,
+  ERROR_INVALID_CONFIG,
+  STATUS_UNAUTHORIZED,
+  STATUS_OK,
+} from './const';
+
+let config = {
+  prepareRefreshTokenRequest: null,
+  shouldIntercept: null,
+  shouldInvalidateAccessToken: null,
+  getAccessTokenFromResponse: null,
+  setRequestAuthorization: null,
```

Comment:
Thought about it, should be more general, what if authorization goes thru query param or any other place? It isn't by the spec, but we probably don't want to be that restrictive, the goal of this method is to authorize request in any way specified. I'm renaming it to `authorizeRequest`.